doc/requirements/management

```diff
@@ -0,0 +1,134 @@
+---
+group: Management
+---
+
+Management operations allows the user to control and maintain Open CAS.
+They are performed from console line with `casadm` management tool.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Remove active cache volume with dirty data with flush
+id: remove_cache_volume_active_flush
+---
+
+While cache volume is active and contains dirty data, the management tool shall
+allow to remove cache volume with flush.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Interrupt removing active cache volume with dirty data with flush
+id: interrupt_removing_cache_volume_active_flush
+---
+
+When active cache volume that contains dirty data is removed with flush,
+the management tool shall allow to interrupt the removal operation
+and the cache volume shall remain its previous state and proper information
+shall be displayed.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Remove active cache volume with dirty data without flush
+id: remove_cache_volume_active_no_flush
+---
+
+While cache volume is active and contains dirty data, the management tool shall
+not allow to remove cache volume without flush without explicitly specify to do
+that by adding _force_ option to command.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Disallow to interrupt removing active cache volume with dirty data
+without flush
+id: interrupt_removing_cache_volume_active_no_flush_disallow
+---
+
+When active cache volume that contains dirty data is removed without flush,
+the management tool shall not allow to interrupt the removal operation and the
+cache volume shall be successfully removed with proper warning about displayed.
```

Comment:
If it is about management operations I would focus on writing requirements for what CAS shall support rather than what it shall not support. Btw with `When active cache volume that contains dirty data is removed without flush (...) the cache volume shall be successfully removed` you are implying that removing dirty core shall always succeed. I think it would be hard to implement